### PR TITLE
added a few lines of poetry

### PR DESCRIPTION
**Pull request by MananDudhwala**
added poetry.txt with a few lines of haiku inside

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request! There are some changes that need to be made in order to format this correctly. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by MananDudhwala** [comment]
added .md in the required folder in the required format

**Review by ltagliaferri — APPROVED** [review]
Thank you for updating this haiku and for contributing to open source 🎉

**Pull request merged**
